install-scripts/mac/skm-install-mac.sh

```diff
@@ -9,13 +9,14 @@ HOME_PATH=~
 INSTALL_PATH="${HOME_PATH}/.splashkit"
 
 function has_git() {
-    git --help 2>&1 > /dev/null
+    echo "Checking for git..."
+    command -v git > /dev/null 2>&1
     return $?
 }
 
 function install_developer_tools() {
-    echo "macOS developer tools are currently not installed which are required for installation skm."
-    read -p "Would you like to install them now? " -n 1 -r < /dev/tty
+    echo "macOS developer tools are currently not installed are required for installation skm."
```

Comment:
**nitpick (non-blocking):** It might read slightly better? 🤔 

```suggestion
    echo "macOS developer tools which are required for skm, are currently not installed."
```